### DISCUSSION TITLE
Neue App: Zwei synchronisierte Karten nebeneinander

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/webgis-community#97] [Ideas]
Hallo!

Ein User (Gutachter) hat uns folgenden Wunsch zugetragen: Zwei Karten nebeneinander auf einem Bildschirm, die den selben Ausschnitt anzeigen. Würde ca. so aussehen:

http://tombatossals.github.io/angular-leaflet-directive/examples/0000-viewer.html#/basic/double-map-access-map-object-example

Oder hier noch ein Beispiel:
https://github.com/jieter/Leaflet.Sync (Beispielkarte: http://jieter.github.io/Leaflet.Sync/examples/dual.html )

Wir halten das für eine interessante App, vor allem Gutachter können so sehr schnell unterschiedliche Zeitstände vergleichen.

Kann man ja mal beim WebGIS Wednesday diskutieren ob es weitere Interessent:innen gibt...

**Comment by gis-eni** [comment]
Dafür gibt es das Live Share Werkzeug.

Wenn du es für den User einfach halten willst, kann man auch APP Template bauen, ähnlich dem "Vergleich/Neben einander" von Luftbildern
